src/Xamarin.Android.Build.Tasks/Utilities/Files.cs

```diff
@@ -198,16 +198,16 @@ public static void ExtractAll(ZipArchive zip, string destination, Action<int, in
 			int i = 0;
 			int total = zip.Entries.Count;
 			foreach (var entry in zip.Entries) {
-
-				if (string.Equals(entry.FullName, "__MACOSX", StringComparison.OrdinalIgnoreCase) ||
-						string.Equals(entry.FullName, ".DS_Store", StringComparison.OrdinalIgnoreCase))
+				if (entry.FullName.Contains ("__MACOSX") ||
```

Comment:
The _MACOSX stuff appears like 

'foo/bar/_MACOSX/.classes.jar'

hence the use of Contains :(